receiver/postgresqlreceiver/testdata/scraper/multiple/expected.json

```diff
@@ -0,0 +1,2240 @@
+                                 },
+                                 {
+                                    "key": "table",
+                                    "value": {
+                                       "stringValue": "public.table2"
+                                    }
+                                 },
+                                 {
+                                    "key": "operation",
+                                    "value": {
+                                       "stringValue": "hot_upd"
+                                    }
+                                 }
+                              ],
+                              "timeUnixNano": "1633639622621536000"
+                           }
+                        ],
+                        "isMonotonic": true
+                     }
+                  },
+                  {
+                     "description": "The number of rollbacks.",
+                     "name": "postgresql.rollbacks",
+                     "sum": {
+                        "aggregationTemporality": "AGGREGATION_TEMPORALITY_CUMULATIVE",
+                        "dataPoints": [
+                           {
+                              "asInt": "2",
+                              "attributes": [
+                                 {
+                                    "key": "database",
+                                    "value": {
+                                       "stringValue": "otel"
+                                    }
+                                 }
+                              ],
+                              "timeUnixNano": "1633639622621536000"
+                           },
+                           {
+                              "asInt": "3",
+                              "attributes": [
+                                 {
+                                    "key": "database",
+                                    "value": {
+                                       "stringValue": "open"
+                                    }
+                                 }
+                              ],
+                              "timeUnixNano": "1633639622621536000"
+                           },
+                           {
+                              "asInt": "4",
+                              "attributes": [
+                                 {
+                                    "key": "database",
+                                    "value": {
+                                       "stringValue": "telemetry"
+                                    }
+                                 }
+                              ],
+                              "timeUnixNano": "1633639622621536000"
+                           }
+                        ],
+                        "isMonotonic": true
+                     }
+                  }
+               ]
+            }
+         ],
+         "resource": {}
+      }
+   ]
+}
```

Comment:
I believe an update was made to the generator function that will add a newline here.